addons/account_edi/models/account_move.py

```diff
@@ -618,7 +618,7 @@ def _get_update_invoice_from_attachment_decoders(self, invoice):
 
     def action_process_edi_web_services(self):
         docs = self.edi_document_ids.filtered(lambda d: d.state in ('to_send', 'to_cancel') and d.blocking_level != 'error')
-        docs._process_documents_web_services()
+        docs._process_documents_web_services(with_commit=False)
```

Comment:
could you check the same in the account_payment file?  @odoostan

Reply:
and I wonder if in action_retry_edi_documents_error as well...

Reply:
Now passing with_commit=False when calling from the tests